dev-docs/bidders/jwplayer.md

```diff
@@ -0,0 +1,257 @@
+---
+layout: bidder
+title: JW Player
+description: Prebid JW Player Bidder Adapter
+biddercode: jwplayer
+gvl_id: 1046
+pbjs: true
+pbs: false
+media_types: video
+userIds: all (with commercial activation)
+tcfeu_supported: true
+dsa_supported: true
+prebid_member: true
+schain_supported: true
+coppa_supported: true
+usp_supported: true
+gpp_supported: true
```

Comment:
Sorry @karimMourra , but we've replaced `gpp_supported` with `gpp_sids` as described in https://docs.prebid.org/dev-docs/bidder-adaptor.html#submitting-your-adapter